gossip/state/state.go

```diff
@@ -639,6 +660,11 @@ func (s *GossipStateProviderImpl) antiEntropy() {
 				logger.Error("Ledger reported block height of 0 but this should be impossible")
 				continue
 			}
+
+			if !s.CanHandleAntiEntropy() {
```

Comment:
Is this check required? It seems that if the peer can't handle antiEntropy then the Go routine shouldn't even be started.

Reply:
Yes, only committer does it.

Note: As discussed yesterday, endorser logic in this scenario has been removed since they weren't working before in cluster run.

Reply:
You are right, will start it only if it is committer

Reply:
done